ush/soca/obs_list_test.yaml

```diff
@@ -0,0 +1,66 @@
+observers:
+- !INC ${OBS_YAML_DIR}/adt_3a_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_3b_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_6a_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_all.yaml
+- !INC ${OBS_YAML_DIR}/adt_c2_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_coperl4.yaml
+- !INC ${OBS_YAML_DIR}/adt_j2_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_j3_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_sa_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/bufr_sfcshipsu.yaml
+- !INC ${OBS_YAML_DIR}/bufr_sfcships.yaml
+- !INC ${OBS_YAML_DIR}/icec_amsr2_north.yaml
+- !INC ${OBS_YAML_DIR}/icec_amsr2_south.yaml
+- !INC ${OBS_YAML_DIR}/icec_emc.yaml
+- !INC ${OBS_YAML_DIR}/icec_nsidc_nh.yaml
+- !INC ${OBS_YAML_DIR}/icec_nsidc_sh.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f17_north.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f17_south.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f18_north.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f18_south.yaml
+- !INC ${OBS_YAML_DIR}/icefb_gdr.yaml
+- !INC ${OBS_YAML_DIR}/salt_bufr_dbuoyprof.yaml
+- !INC ${OBS_YAML_DIR}/salt_bufr_mbuoybprof.yaml
+- !INC ${OBS_YAML_DIR}/salt_profile_fnmoc_godas.yaml
+- !INC ${OBS_YAML_DIR}/salt_profile_fnmoc.yaml
+- !INC ${OBS_YAML_DIR}/salt_profile_wod.yaml
+- !INC ${OBS_YAML_DIR}/sss_smap_jpl.yaml
+- !INC ${OBS_YAML_DIR}/sss_smap.yaml
+- !INC ${OBS_YAML_DIR}/sss_smos_esa.yaml
+- !INC ${OBS_YAML_DIR}/sss_smos.yaml
+- !INC ${OBS_YAML_DIR}/sss_trak_fnmoc.yaml
+- !INC ${OBS_YAML_DIR}/sst_amsr2_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_drifter.yaml
+- !INC ${OBS_YAML_DIR}/sst_gmi_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_metopa_l3u_so025.yaml
+- !INC ${OBS_YAML_DIR}/sst_metopb_l3u_so025.yaml
+- !INC ${OBS_YAML_DIR}/sst_metopc_l3u_so025.yaml
+- !INC ${OBS_YAML_DIR}/sst_noaa18_l3u_so025.yaml
+- !INC ${OBS_YAML_DIR}/sst_noaa19_l3u_so025.yaml
+- !INC ${OBS_YAML_DIR}/sst_noaa19_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_ostia.yaml
+- !INC ${OBS_YAML_DIR}/sst_ship_fnmoc.yaml
+- !INC ${OBS_YAML_DIR}/sst_trak_fnmoc.yaml
+- !INC ${OBS_YAML_DIR}/sst_viirs_n20_l3u_so025.yaml
+- !INC ${OBS_YAML_DIR}/sst_viirs_npp_l3u_so025.yaml
+- !INC ${OBS_YAML_DIR}/sst_windsat_l3u.yaml
+- !INC ${OBS_YAML_DIR}/temp_bufr_dbuoyprof.yaml
+- !INC ${OBS_YAML_DIR}/temp_bufr_mbuoybprof.yaml
+- !INC ${OBS_YAML_DIR}/temp_profile_fnmoc_godas.yaml
+- !INC ${OBS_YAML_DIR}/temp_profile_fnmoc.yaml
+- !INC ${OBS_YAML_DIR}/temp_profile_wod.yaml
+- !INC ${OBS_YAML_DIR}/insitu_argo_profile_YYYYMMDD.yaml
```

Comment:
please remove YYYYMMDD from the insitu yaml files:
Lines 53-59.

Reply:
done